components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -3038,4 +3038,19 @@ private boolean isGroup(String localRoleName) {
         return !Stream.of(INTERNAL_DOMAIN, APPLICATION_DOMAIN, WORKFLOW_DOMAIN).anyMatch(domain -> localRoleName
                 .toUpperCase().startsWith((domain + UserCoreConstants.DOMAIN_SEPARATOR).toUpperCase()));
     }
+
+    /**
+     * Get tenant context using tenant domain.
+     *
+     * @param tenantDomain tenant domain of the tenant
+     * @return tenant context
+     */
+    private String getTenantContextFromTenantDomain(String tenantDomain) {
+
+        if (!MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equalsIgnoreCase(tenantDomain)) {
+            return MultitenantConstants.TENANT_AWARE_URL_PREFIX + "/" + tenantDomain + "/";
```

Comment:
What's the value of `MultitenantConstants.TENANT_AWARE_URL_PREFIX`?

Reply:
"t"